test/helper.rb

```diff
@@ -74,6 +74,10 @@ def self.rbx?
       defined? RUBY_ENGINE and 'rbx' == RUBY_ENGINE
     end
 
+    def self.ruby_22?
+      defined? RUBY_ENGINE and 'ruby' == RUBY_ENGINE and /^2\.2\.\d+/ =~ RUBY_VERSION
```

Comment:
You should do `RUBY_VERSION > '2.2'` and you don't need any RUBY_ENGINE checks. We want this method to return true for every version from 2.2.0 forward. We don't want Typhoeus tests run on 2.3.